src/Faker/Provider/Base.php

```diff
@@ -156,7 +156,7 @@ public static function randomAscii()
     }
 
     /**
-     * Returns random elements from a provided array
+     * Returns random elements from a provided array (not repeating input indices)
```

Comment:
What about this?
> Returns randomly ordered subsequence of $count elements from a provided array

I want for the other users to be clear that the function does not repeat input elements. The current comment does not say it.